src/codec-msbc.c

```diff
@@ -114,10 +114,7 @@ int msbc_init(struct esco_msbc *msbc) {
 	msbc->seq_number = 0;
 	msbc->frames = 0;
 
-	/* Initialize PLC context. When calling with non-NULL parameter,
-	 * this function does not allocate anything - there is no need
-	 * to call plc_free(). */
-	plc_init(&msbc->plc);
+	msbc->plc = plc_init(NULL);
```

Comment:
The `msbc_init` might be called more than once without calling `msbc_finish`. In order not to leak memory you have to free plc context if it is not NULL before allocating new one.

Reply:
I'll just use `msbc_init` on the already-allocated pointer, even if it theory, `msbc_release` should probably be used (it does not do anything).